src/sites/twitch-twilight/modules/directory/following.js

```diff
@@ -432,13 +432,12 @@ export default class Following extends SiteModule {
 					const channelCardTitle = card.querySelector('.ffz-channel-data .live-channel-card__title');
 
 					const textContent = hostObj.channels.length > 1 ? `${hostObj.channels.length} hosting ${displayName}` : inst.props.title;
-					if (channelCardTitle !== null) {
-						channelCardTitle.textContent
-							= channelCardTitle.title
-							= textContent;
-					}
+					const title = hostObj.channels.length > 1 ? `${hostObj.channels.join(', ')} hosting ${displayName}` : inst.props.title;
```

Comment:
Left this as `inst.props.title` for consistency in case that changes on the twitch side.